swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    
+    def __init__(self, category = "Clothing", condition = 0.0):
+        self.category = category
+        self.condition = condition
```

Comment:
There are two issues here, first, this setup makes it possible to pass in a value for category that is not "Clothing", which could result in an instance of Clothing that has some other category.  Here's a fix for this issue:
```suggestion
    def __init__(self, condition = 0.0):
        self.category = "Clothing"
        self.condition = condition
```
The second issue is that Clothing is a subclass of Item, and the constructor for Item can do this work for you:
```suggestion
    def __init__(self, condition = 0.0):
        super().__init__("Clothing", condition)
```